CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.5.3]
+### Changed
+* The iterative method in `flood map` now iterates once using the `nmad` method as an initial guess and runs with a maximum step size of 3. 
+
+## Added
+* A `minimization_metric` parameter for `flood_map.iterative` with a default value of `fmi`.
```

Comment:
```suggestion
* You can choose whether an `fmi`  or `ts` minimization metric is used for the `flood_map.iterative` method:
  * the `flood_map` console script entrypoint now accepts a `--min-metric` argument
  * the  `floopd_map.make_flood_map` function now accepts a `min_metric` keyword argument
```

Reply:
Also I generally prefer the variable name/argument to be `minimization_metric` (as you have it in the changelog) instead of `min_metric` but `min_metric` is *fine*.